projects/phone/src/lib/plugins/add-on-focus.ts

```diff
@@ -0,0 +1,11 @@
+import {MaskitoPlugin} from '@maskito/core';
+import {maskitoEventHandler} from '@maskito/kit';
+
+export function maskitoAddOnFocusPlugin(value: string): MaskitoPlugin {
```

Comment:
Should we move `maskitoAddOnFocusPlugin` / `maskitoRemoveOnBlurPlugin` to `@maskito/kit` ? 🤔

Reply:
I wanted to suggest the same